src/videojs-http-streaming.js

```diff
@@ -829,6 +831,11 @@ class HlsHandler extends Component {
       delete this.tech_.hls;
     }
 
+    if (this.mediaSourceUrl_ && window.URL.revokeObjectURL) {
+      window.URL.revokeObjectURL(this.mediaSourceUrl_);
```

Comment:
Alternatively, we could've gotten the blob url via `this.tech_.src()` but I think this is fine as well.